website/pages/docs/developers/creating-new-plugin/go-source.mdx

```diff
@@ -54,6 +54,14 @@ Some APIs lend themselves to being synced incrementally: rather than fetch all p
 
 Incremental tables are great for efficiency, but add some additional complexity both on you and on your users. As the plugin author, you should consider first whether the table needs to be incremental, then whether it can be made to be incremental. You can also [read more about managing incremental tables](/docs/advanced-topics/managing-incremental-tables).
 
+Here are the basic steps of adding support for incremental tables to your plugin:
+
+1. Within the plugin client initialize the state client using `state.NewConnectedClient` or `state.NewConnectedClientWithOptions` ([see example in our Hacker News plugin](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/hackernews/resources/plugin/client.go)).
```

Comment:
I would link to the exact lines that you are referencing

Reply:
Done in [eb7c2a8](https://github.com/cloudquery/cloudquery/pull/17824/commits/eb7c2a820ee6e9a8cd42ff6d4657b712839e1f76)